ui/app/components/actions-flyout-global-button.hbs

```diff
@@ -0,0 +1,21 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
```

Comment:
This happy little button that pops out when you have your fly-out closed:
![image](https://github.com/hashicorp/nomad/assets/713991/a41abefd-c56e-4ece-b1f9-2b9e9b364fa1)